playground/optimize-deps/vite.config.js

```diff
@@ -1,6 +1,9 @@
 const fs = require('fs')
 const vue = require('@vitejs/plugin-vue')
 
+// Prevent vitest from setting process.env.NODE_ENV
```

Comment:
Actually, this is not preventing. This is overriding it.
My understanding is:

1. Vitest starts up
2. Vitest sets `process.env.NODE_ENV` to `'test'` because `process.env` is not set
3. `vite dev`/`vite build` runs
4. Vite loads config
5. this line gets executed and sets `process.env.NODE_ENV` to `''`
6. Vite runs dev/build

Reply:
You are right!